bakery/src/scripts/Dockerfile

```diff
@@ -4,7 +4,7 @@ LABEL maintainer="OpenStax Content Engineering"
 
 RUN apt-get update
 RUN apt-get install -y build-essential libicu-dev pkg-config libmagic1 mime-support wget curl xsltproc lsb-release git
-RUN apt-get install -y imagemagick icc-profiles-free
+RUN apt-get install -y imagemagick icc-profiles-free curl unzip
```

Comment:
Minor: I just happened to notice this (and it's not directly related to this PR anyways), but is there some reason why we have two consecutive `apt-get install` steps here in the `Dockerfile`?

Reply:
I was not aware about that. Just informed myself now about [Docker best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run) and updated the line.